resources/CMakeLists.txt

```diff
@@ -0,0 +1,26 @@
+set(RESOURCES_FILES
+  Fonts/Gayathri-Regular.ttf
+  Fonts/Raleway-Regular.ttf
+
+  Images/novel-chan.png
+
+  Scripts/avg.lua
+
+  Shaders/BasicFragmentShader.glsl
+  Shaders/BasicVertexShader.glsl
+  Shaders/FontFragmentShader.glsl
+  Shaders/FontVertexShader.glsl
+  Shaders/TexturedFragmentShader.glsl
+  Shaders/TexturedVertexShader.glsl
+)
+
+foreach(resource ${RESOURCES_FILES})
+  configure_file(${resource} Resources/${resource} COPYONLY)
+endforeach()
```

Comment:
Does this copy the files whenever NovelRT itself is targeted as a dependency by something else? I remember this command being the bane of my existence the last time I tried it.

Reply:
It copies them to CMAKE_CURRENT_BINARY_DIR, and then targets can copy using the resources target into their own source/binary trees.